api/src/main/java/javax/enterprise/inject/spi/BeanManager.java

```diff
@@ -618,4 +620,18 @@
      */
     <T> InterceptionFactory<T> createInterceptionFactory(CreationalContext<T> ctx, Class<T> clazz);
 
+
+    /**
+     *
+     * Obtains an {@link Instance} giving access to beans instances
+     *
+     * Instances of dependent scoped beans obtained with this Instance should be explicitly release with {@link Instance#destroy(Object)}
+     *
```

Comment:
We should also mention that only beans which are eligible for injection into the class into which the `BeanManager` was injected are accessible from the returned `Instance`. Something similar to `javax.enterprise.inject.spi.BeanManager.getBeans(Type, Annotation...)` javadoc.